administrator/components/com_media/resources/styles/components/_media-infobar.scss

```diff
@@ -11,6 +11,10 @@
   background-color: $info-bg;
   border-left: 1px solid $border-color;
   h2 {
```

Comment:
@proman24 Would you like to make the changes here in this PR?.. 

```
word-wrap: break-word;
padding: 10px 40px 10px 15px;
```

Reply:
@ciar4n Sure, would be updating this PR soon.